src/Microsoft.Sbom.Api/Utils/Constants.cs

```diff
@@ -47,6 +47,11 @@ public static class Constants
         SPDX30ManifestInfo
     };
 
+    public static Collection<ComplianceStandardType> SupportedComplianceStandards = new()
+    {
+        ComplianceStandardType.NTIA,
```

Comment:
Does this need to include `ComplianceStandardType.None` to accept "-cs None" on teh command line?